src/Core/MySQL/MySQLJsonConv.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
```

Comment:
Json is named in a different style than MySQL, and it feels wrong - let's rename it to JSON, so it will be `MySQLJSONConversion.h`